File: Labs/Lab 2/Lab 2.java
import java.util.Arrays;

public class Anagram {
    public boolean isAnagram(String s, String t) {
        if (s.length() != t.length()) {
            return false;
        }
        
        char[] arr = s.toCharArray();
        char[] arr1 = t.toCharArray();
        Arrays.sort(arr);
        Arrays.sort(arr1);
        
        return Arrays.equals(arr, arr1);
    }
    public static void main(String[] args) {
        Anagram anagram = new Anagram();

        System.out.println(anagram.isAnagram("listen", "silent")); 
        System.out.println(anagram.isAnagram("rat", "car")); 
    }
}

File: Labs/Lab 2/Lab 2.py
class Anagram:
    def isAnagram(self, s: str, t: str) -> bool:
        return sorted(s)==sorted(t)

def main():
    anagram = Anagram()

    # Test cases
    print(anagram.isAnagram("listen", "silent"))  
    print(anagram.isAnagram("rat", "car"))       

if __name__ == "__main__":
    main()
